crates/pallet-feeds/src/mock.rs

```diff
@@ -1,3 +1,6 @@
+// Silence a compiler warning in `construct_runtime!`
+#![allow(non_camel_case_types)]
```

Comment:
For such awkward things please add TODO explaining why we silence it with link to upstream issue. Right now it looks like compiler needs it, but it is not actually the case and having documentation explaining when it can be removed it helpful.

Reply:
Force-pushed in commit 693de5ae4c